aptos-move/block-executor/src/task.rs

```diff
@@ -80,11 +78,7 @@ pub trait ExecutorTask: Sync {
     /// Execute a single transaction given the view of the current state.
     fn execute_transaction(
         &self,
-        view: &impl TExecutorView<
-            <Self::Txn as Transaction>::Key,
-            MoveTypeLayout,
-            <Self::Txn as Transaction>::Identifier,
-        >,
+        view: &impl TExecutorView<<Self::Txn as Transaction>::Key, MoveTypeLayout, AggregatorID>,
```

Comment:
I commented on @igor-aptos 's PR that it might be okay to specialize Block executor on AggregatorID, at least for now, but want to make sure we are consistent with the decision on this?

Reply:
Specializing is good, I had it before but then removed because of the other PRs.